src/lib.rs

```diff
@@ -162,6 +162,30 @@ where
     /// Collect range into a container
     ///
     /// Works for any container type that implements [`FromIterator`].
+    ///
+    /// # Examples
+    ///
+    /// Basic usage, creating a [`Vec`]. See also [`Range::to_vec`]
+    ///
+    /// ```rust
+    /// extern crate a_range;
+    ///
+    /// let collection = a_range::from(42).up_to(45).collect::<Vec<_>>();
```

Comment:
I kinda prefer to write it with the type annotation instead of the turbofish, but you don't need to change it :)

```suggestion
    /// let collection: Vec<_> = a_range::from(42).up_to(45).collect();
```